lab-justin/gulpfile.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const gulp = require('gulp');
+const eslint = require('gulp-eslint');
+const mocha = require('gulp-mocha');
+const nodemon = require('gulp-nodemon');
+var paths = ['**/*.js', 'lib/*.js', 'model/*.js', 'test/*.js', 'route/*.js'];
+
+gulp.task('mocha', function(){
+  return gulp.src(paths)
+  .pipe(mocha());
+});
+
+gulp.task('eslint', function(){
+  return gulp.src(paths)
+  .pipe(eslint())
+  .pipe(eslint.format())
+  .pipe(eslint.failAfterError());
+});
+
+gulp.task('test', function(){
```

Comment:
This file doesn't exist, so your test task does nothing. As a result, it dodges everything that we're trying to set up with TravisCI. I'll ask for a resubmit for any future assignments turned in like this.